lib/http/HttpConnectionsStream.js

```diff
@@ -35,6 +35,7 @@ HttpConnectionsStream.prototype.close = function () {
 
 HttpConnectionsStream.prototype._fetchNextPage = function () {
   var self = this;
+  this.emit('request');
```

Comment:
Forgot the first page. Tried to put it in HttpFetcher by extending EventEmitter, but got `undefined is not a function` when registering an eventlistener in Fetcher.js: 
```javascript
this.http = new HttpFetcher(20);
this.http.on('request', function() { ... });
```